Ktisis/Interface/Windows/Workspace/Tabs/PoseTab.cs

```diff
@@ -177,59 +178,10 @@ public static class PoseTab {
 				KtisisGui.FileDialogManager.OpenFileDialog(
 					"Importing Pose",
 					"Pose Files (.pose){.pose}",
-					(success, path) => {
+					(success, paths) => {
```

Comment:
Resolved in 2025f82.

Reply:
Just for context, the reason I did rename was because it was a `List<string>` and it made it a little clearer overall to myself.
No biggie either way!